daemon/containerd/image_list.go

```diff
@@ -132,6 +132,7 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 					return nil, nil
 				}
 				if !available || len(missing) > 0 {
+					println("available", available, "missing", missing)
```

Comment:
Looks like debugging code 😅

Reply:
It was! I was debugging something and wanted to make sure I didn't forget, but it was unrelated 😅